File: app/src/main/java/com/rowland/delivery/features/auth/di/components/AuthComponent.java
package com.rowland.delivery.features.auth.di.components;

import com.rowland.delivery.features.auth.ui.AuthActivity;
import com.rowland.delivery.services.auth.di.modules.AuthModule;

import javax.inject.Singleton;

import dagger.Component;

/**
 * Created by Rowland on 5/1/2018.
 */

@Component(modules = AuthModule.class)
@Singleton
public interface AuthComponent {
    void injectAuthActivity(AuthActivity activity);
}

File: ui/src/main/java/com/rowland/delivery/merchant/features/auth/di/components/AuthComponent.kt
package com.rowland.delivery.merchant.features.auth.di.components


import com.rowland.delivery.merchant.features.auth.di.modules.AuthModule
import com.rowland.delivery.merchant.features.auth.ui.AuthActivity
import dagger.Component
import javax.inject.Singleton

/**
 * Created by Rowland on 5/1/2018.
 */

@Component(modules = arrayOf(AuthModule::class))
@Singleton
internal interface AuthComponent {
    fun injectAuthActivity(activity: AuthActivity)
}
